docs/release/3.4/release-3-4-24.md

```diff
@@ -0,0 +1,132 @@
+OSG Software Release 3.4.24
+===========================
+
+**Release Date**: 2019-02-21
+
+Summary of changes
+------------------
+
+This release contains:
+
+-   BLAHP 1.18.39: Now propagates signals down to the payload job
+-   osg-pki-tools 3.0.1: Now accepts multi-word states or provinces
+-   HTCondor-CE 3.2.1: Minor adjustments to work with new HTCondor versions (8.6.x, 8.8.x, and 8.9.x)
+-   condor-cron 1.1.4: Minor adjustments to work with new HTCondor versions (8.6.x, 8.8.x, and 8.9.x)
```

Comment:
None of the changes for the above were necessary for 8.6.x

Reply:
I wanted to express the fact that both the old and new versions work with the change.

Reply:
Dropping parenthetical